react/modules/tasks/class-components.md

```diff
@@ -195,7 +279,11 @@ Fetch the full list (or first page) of items from the API.
 
 ### ❓ How to simulate a failed request?
 
-Use ModResponse or set an invalid endpoint to trigger a 4xx/5xx response.
+Use MockResponse or set an invalid endpoint to trigger a 4xx/5xx response.
```

Comment:
It should be ModResponse
https://chromewebstore.google.com/detail/modresponse-mock-and-repl/bbjcdpjihbfmkgikdkplcalfebgcjjpm